blueocean-pipeline-api-impl/src/test/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineBaseTest.java

```diff
@@ -52,12 +54,18 @@
 public abstract class PipelineBaseTest{
     private static  final Logger LOGGER = LoggerFactory.getLogger(PipelineBaseTest.class);
 
-    public PipelineBaseTest() {
+    @BeforeClass
```

Comment:
this may fix a few flaky tests (if caused by JWT being enabled when not expected)